### PR TITLE
minor updates on editor.xml

### PR DESCRIPTION
**Pull request by t4x0n**
a mistake in line 82:  word "Resgistrar" is changed for "Registrar"

and "Fichero" is changed for "Archivo", in Spain they use "Fichero" and "Archivo", but in general "Archivo" is better 

I proposed this change, and I can be made other changes similar if your approve that!

### CODE REVIEW

**Comment by mtub** [comment]
@marcbria, could you have a look?

File: locale/es_ES/editor.xml [hunk 1]
```diff
@@ -100,23 +100,23 @@
...
 	<message key="editor.submission.proofreading.confirmRemoveCompletion">Anula la aceptación de esta prueba para indicar que la revisión ya no está completa y que el fichero no está listo para ser publicado.</message>
-	<message key="editor.submission.copyedit.manageCopyeditFilesDescription">Cualquier fichero que ya haya sido subido en cualquier fase del envío puede añadirse a la lista de ficheros de corrección marcando la siguiente casilla "Añadir" y pulsando "Buscar": todos los ficheros disponibles serán listados y podrán incluirse. Los ficheros adicionales que todavía no se hayan subido a esta fase se pueden añadir al listado pulsando el enlace "Cargar fichero".</message>
+	<message key="editor.submission.copyedit.manageCopyeditFilesDescription">Cualquier archivo que ya haya sido subido en cualquier fase del envío puede añadirse a la lista de ficheros de corrección marcando la siguiente casilla "Añadir" y pulsando "Buscar": todos los ficheros disponibles serán listados y podrán incluirse. Los ficheros adicionales que todavía no se hayan subido a esta fase se pueden añadir al listado pulsando el enlace "Cargar fichero".</message>
```
Comment (marcbria) [inline locale/es_ES/editor.xml]: You missed one "fichero" at the end.
Thanks for the contrib.

File: locale/es_ES/editor.xml [hunk 2]
```diff
@@ -100,23 +100,23 @@
...
 	<message key="submission.queries.messageRequired">Es necesario especificar un mensaje de discusión.</message>
-	<message key="submission.queries.attachedFiles">Ficheros adjuntos</message>
+	<message key="submission.queries.attachedFiles">Archivo adjuntos</message>
 	<message key="grid.action.editQuery">Editar discusión</message>
-	<message key="editor.submission.query.manageQueryNoteFilesDescription">Puede adjuntar ficheros para incluirlos en esta discusión desde la siguiente lista.</message>
+	<message key="editor.submission.query.manageQueryNoteFilesDescription">Puede adjuntar archivos para incluirlos en esta discusión desde la siguiente lista.</message>
 	<message key="editor.submission.selectedReviewer">Revisor seleccionado</message>
 	<message key="editor.submission.proofreading.approveProof">Aceptar la prueba</message>
 	<message key="editor.submission.proofreading.revokeProofApproval">Anular la aceptación de la prueba</message>
 	<message key="editor.submission.proofreading.confirmCompletion">Acepta esta prueba para indicar que la revisión se ha completado y que el fichero está listo para ser publicado.</message>
```
Comment (marcbria) [inline locale/es_ES/editor.xml]: change this too. please

File: locale/es_ES/editor.xml [hunk 3]
```diff
@@ -100,23 +100,23 @@
...
 	<message key="editor.submission.proofreading.confirmRemoveCompletion">Anula la aceptación de esta prueba para indicar que la revisión ya no está completa y que el fichero no está listo para ser publicado.</message>
```
Comment (marcbria) [inline locale/es_ES/editor.xml]: missed this "fichero"

**Comment by t4x0n** [comment]
thanks @marcbria the Google Chrome search is not enough good, and I am new in GitHub, now I know the GitHub Search ;)

**Comment by marcbria** [comment]
Gracias @t4x0n por la contribución y los cambios.
@mtub you can merge this too.

**Pull request merged**